test/writehero_test.cpp

```diff
@@ -306,7 +306,6 @@ static void AssertPlayer(PlayerStruct &player)
 	ASSERT_EQ(player._pAblSpells, 134217728);
 	ASSERT_EQ(player._pScrlSpells, 0);
 	ASSERT_EQ(player._pSpellFlags, 0);
-	ASSERT_EQ(player._pwtype, 1);

```

Comment:
assert true UsesRangedWeapon() to maintain coverage and test your new code.

Reply:
done 🙂